ytmapi-rs/Cargo.toml

```diff
@@ -22,3 +22,8 @@ chrono = "0.4.31"
 
 [dev-dependencies]
 pretty_assertions = "1"
+
+[features]
```

Comment:
please add "default" feature that uses reqwest defaults (breaks youtui binary otherwise)